frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-items.tsx

```diff
@@ -334,6 +334,8 @@ const OperatorHubTile: React.FC<OperatorHubTileProps> = ({ item, onClick }) => {
   const badges = ([
```

Comment:
I believe this could be simpler so that any item with `catalogSourceDisplayName` gets the badge.
```suggestion
  const badges = item?.catalogSourceDisplayName
    ? [<Badge text={item.catalogSourceDisplayName} />]
    : [];
```

Reply:
thanks, updated